examples/vscode-sandbox/kro/sigs_sandbox_70.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: custom.agents.x-k8s.io/v1alpha1
```

Comment:
Is this example related to this PR? Seems better to keep them separate

Reply:
no. removed.